reference/general/README.md

```diff
@@ -156,38 +147,21 @@ For running a node cluster of load balanced Lotus JSON RPC API endpoints.
 ### Storage provider tools
 
 * [Bidbot](https://github.com/textileio/bidbot) - A Filecoin Network sidecar to bid in storage deal auctions.
```

Comment:
```suggestion
```
Bidbot is no longer maintained and it's readme points to estuary et al, so lets definitely remove it here as well to prevent more confusion.